src/pages/Swap.tsx

```diff
@@ -163,21 +162,21 @@ function Swap(): ReactElement {
         ? (
             formState.currentSwapPairs
               .map(({ to, type: swapType }) => {
-                if (!TOKENS_MAP?.[to.symbol]) {
+                if (!tokensMap?.[to.symbol]) {
```

Comment:
```
const token = tokensMap?.[to.symbol]
if (!token) log and return null
```
now token is guaranteed to be defined and you can skip all of the `token?.name ?? ""` again